File: translation_output/w_test/gpt-4/codenet/java/python/p02717.py
Certainly! Below is the Python code that achieves the same functionality as the provided Java code:

```python
def rotate_values(input_string):
    values = input_string.split()
    a, b, c = values
    return f"{c} {a} {b}"

# Sample test case
input_string = "1 2 3"
output = rotate_values(input_string)
print(output)
```

This Python function `rotate_values` takes a string `input_string` containing three integers separated by spaces, splits the string into three variables, and then returns them in rotated order, formatted as a string.

For your given test input "1 2 3", calling the `rotate_values` function with this input and printing the result will give you the expected output "3 1 2".

File: translation_output/w_test/gpt-4/codenet/python/java/p02717.java
Here's the equivalent Java code for the Python code snippet you provided:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Read space-separated integers
        int X = scanner.nextInt();
        int Y = scanner.nextInt();
        int Z = scanner.nextInt();

        // Swap the values
        int A = Z;
        int B = X;
        int C = Y;

        // Print the result
        System.out.println(A + " " + B + " " + C);
        
        scanner.close();
    }
}
```

To test the Java code with the input `1 2 3`, you would compile and run the `Main` class, enter `1 2 3` when prompted, and expect the output `3 1 2`.
